src/Processors/Formats/Impl/PrometheusTextOutputFormat.cpp

```diff
@@ -107,17 +108,25 @@ void PrometheusTextOutputFormat::fixupBucketLabels(CurrentMetric & metric)
 {
     String bucket_label = metric.type == "histogram" ? "le" : "quantile";
 
-    std::sort(metric.values.begin(), metric.values.end(),
+    ::sort(metric.values.begin(), metric.values.end(),
         [&bucket_label](const auto & lhs, const auto & rhs)
         {
+            bool lhs_labels_contain_sum = lhs.labels.contains("sum");
+            bool lhs_labels_contain_count = lhs.labels.contains("count");
+            bool lhs_labels_contain_sum_and_count = lhs_labels_contain_sum || lhs_labels_contain_count;
```

Comment:
The right side has ||. Should the variable be called `lhs_labels_contain_sum_or_count`?

Reply:
Yes sure.